src/packages/migrate/src/utils/getGithubIssueUrl.ts

```diff
@@ -0,0 +1,82 @@
+import newGithubIssueUrl from 'new-github-issue-url'
+import open from 'open'
+import prompt from 'prompts'
+import stripAnsi from 'strip-ansi'
+
+export function getGithubIssueUrl({
+  title,
+  user = 'prisma',
+  repo = 'migrate',
+  template = 'bug_report.md',
+  body,
+}: {
+  title: string
+  user?: string
+  repo?: string
+  template?: string
+  body?: string
+}): string {
+  return newGithubIssueUrl({
+    user,
+    repo,
+    template,
+    title,
+    body,
+  })
+}
+interface IssueOptions {
+  title?: string,
+  error: any,
+  cliVersion: string,
+  binaryVersion: string,
+  reportId?: number,
+}
+export async function wouldYouLikeToCreateANewIssue(
+  options: IssueOptions
+) {
+  const shouldCreateNewIssue = await prompt({
+    type: 'select',
+    name: 'value',
+    message: 'To help us even more, would you like to create an issue',
```

Comment:
```suggestion
    message: 'To help us even more, would you like to create an issue?',
```

Reply:
So first question is why to we ask? 
Because we could just generate the link automatically and let the user know "help us by creating a GitHub issue using this link"

Reply:
Because it automatically opens the issue in your default browser and the generated link is very long and ugly